### PR TITLE
Extend `is_invertible_with_inverse` to rings of type `MPolyQuoRing{<:MPolyDecRingElem}`

### PR DESCRIPTION
**Pull request by wdecker**
…MPolyDecRingElem}`

### CODE REVIEW

**Comment by wdecker** [comment]
I added an example from intersection theory where the extension is needed.

File: src/Rings/MPolyQuo.jl [hunk 1]
```diff
@@ -1100,6 +1100,33 @@ end
...
+
+julia> f = A(c[1]^2 - c[1] - c[2] + 1)
+c[1]^2 - c[1] - c[2] + 1
+
+julia> inv(f)
+c[1] + c[2] + c[3] + 1
+
+julia> f*inv(f)
+1
+
```
Comment (joschmitt) [inline src/Rings/MPolyQuo.jl]: Somehow the documented function is not mentioned in this example?

Comment (wdecker) [reply src/Rings/MPolyQuo.jl]: Thx, I confused two versions of what I had. Should be o.k. now.I will add more of this to the docu a little later today.

**Comment by wdecker** [comment]
O.k. from my side if lights are green

**Review by fieker — APPROVED** [review]

**Pull request merged**